src/terrain/clients/notifications.clj

```diff
@@ -79,9 +82,22 @@
                       :subject "Added to team"
                       :email true
                       :email_template "added_to_team"
-                      :payload {:email_address (:email user)
+                      :payload {:action "added_to_team"
```

Comment:
I also added a `payload.action` to each Team notification, since the UI seems to be looking for an `added_to_team` action, but it's currently checking the `email_template` instead.